Makefile

```diff
@@ -196,8 +196,8 @@ CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
 ENVTEST ?= $(LOCALBIN)/setup-envtest
 
 ## Tool Versions
-KUSTOMIZE_VERSION ?= v5.0.1
-CONTROLLER_TOOLS_VERSION ?= v0.12.0
+KUSTOMIZE_VERSION ?= v5.4.2
+CONTROLLER_TOOLS_VERSION ?= v0.15.0
```

Comment:
Would it be possible to move this commit in a separate pull request?

Reply:
https://github.com/checkpoint-restore/checkpoint-restore-operator/pull/21

Reply:
PR #21 has been merged and this commit can be dropped.